performanceMetrics/package.json

```diff
@@ -30,16 +30,16 @@
 		"dist/*"
 	],
 	"dependencies": {
-		"@enact/core": "^4.5.2",
-		"@enact/i18n": "^4.5.2",
-		"@enact/sandstone": "^2.5.2",
-		"@enact/spotlight": "^4.5.2",
-		"@enact/ui": "^4.5.2",
-		"axios": "^0.24.0",
-		"classnames": "^2.2.5",
+		"@enact/core": "^4.6.0",
+		"@enact/i18n": "^4.6.0",
+		"@enact/sandstone": "^2.6.0",
+		"@enact/spotlight": "^4.6.0",
+		"@enact/ui": "^4.6.0",
+		"axios": "^1.2.5",
```

Comment:
-> Could you share breaking changes?
https://github.com/axios/axios/releases/tag/v1.0.0

> Deprecated
There are multiple deprecations, refactors and fixes provided in this release. Please read through the full release notes to see how this may impact your project and use case.

>Removed
Removed incorrect argument for NetworkError constructor [#4656](https://github.com/axios/axios/pull/4656)
Removed Webpack [#4596](https://github.com/axios/axios/pull/4596)
Removed function that transforms arguments to array [#4544](https://github.com/axios/axios/pull/4544)

I found this and feel a little worried about that.

Reply:
@hong6316 , in performanceMetrics, we only use the basic axios.get(). The list of breaking changes for version 1.0.0 does not include anything about this function. I tested the app locally and there were no errors
![image](https://user-images.githubusercontent.com/63335068/220570612-c72c8130-ee98-4b27-a84f-0a03544142d3.png)
The link you shared is correct, but I don't see anything affection our app

Reply:
:+1:

Reply:
@daniel-stoian-lgp short question, what if we use fetch() instead of axios? If we use only axios.get(), I don't see the reason why we should use axios.

Reply:
@seunghoh, it's a good idea. I will put this in the TODO list